portal-api/src/v1/gef/models/facilities.js

```diff
@@ -31,6 +31,9 @@ class Facility {
       this.coverEndDate = null;
       this.issueDate = null;
       this.monthsOfCover = null;
+      this.facilityEndDateExists = null;
+      this.facilityEndDate = null;
```

Comment:
Should a version 0 deal have `facilityEndDate: null` on it? I don't think it really makes a difference but it's maybe a bit surprising

Reply:
Could you add a TODO here for 7171 & I'll update these (I'm already covering facilityEndDateExists)

Reply:
Good point, I'll add it as a TODO

Reply:
Added 👍